README-HE.md

```diff
@@ -188,7 +187,7 @@
 ## רישיון
 <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by-sa/4.0/88x31.png" /></a><br />פרוייקט זה נכתב תחת רישיון 
 
-<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
```

Comment:
This can be kept in and added back.

Reply:
done add66b602f87077fb9a28bfc9c24e09203be26b5